lua/packer/plugin_utils.lua

```diff
@@ -284,4 +284,12 @@ plugin_utils.post_update_hook = function(plugin, disp)
   end)
 end
 
+plugin_utils.post_install_hook = function(plugin, disp)
+  return run_hook(plugin, disp, 'post_install')
```

Comment:
Rather than an ever expanding list of arguments, I think when you start to pass more and more args into a function it might be time to pass in a table instead, so you scale out the inputs more easily without worrying about naked args and positioning and nil-ness i.e.
I think this should be
```lua
  return run_hook(plugin, disp, { hook_name = 'post_install' })
```

Reply:
Not sure if it needs it as it's an internal function but I've left the default `hook_name` as an empty string. Not sure what the best thing to set it to would be?